### PR TITLE
feat: add_about-govtech _page

### PR DESCRIPTION
**Pull request by PonlokTTR**
add about-govtech_page

### CODE REVIEW

File: components/about-govtech/about-govtech.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Button } from "../ui/button";
+import Text from "../ui/text";
+
+export default function AboutGovtech() {
+  return (
+    <div className=" mx-auto mt-5 p-5 md:flex">
+      <div className="max-w-md ">
+        <Text variant="heading" className="text-center text-[#0F45A8]">
```
Comment (Visethboti) [inline components/about-govtech/about-govtech.tsx:8]: This should subheading

File: components/about-govtech/about-govtech.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+          Ipsum passages, and more recently with desktop publishing software like Aldus
+          PageMaker including versions of Lorem Ipsum.
+        </Text>
+        <Text variant="smallheading" className="mt-12">
+          March 14, 2024
+        </Text>
+        <Text variant="smallheading" className="mt-5">
+          It is a long established fact that a reader will be distracted by the readable content
+          of a page when looking at its layout
+        </Text>
```
Comment (Visethboti) [inline components/about-govtech/about-govtech.tsx:22]: These should be body

File: components/about-govtech/about-govtech.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import Text from "../ui/text";
+
+export default function AboutGovtech() {
+  return (
+    <div className=" mx-auto mt-5 p-5 md:flex">
+      <div className="max-w-md">
+        <Text variant="subheading" className="text-[#0F45A8]">
+          About Govtech
+        </Text>
+        <Text variant="body" className="mt-12 max-w-96">
```
Comment (avevotsira) [inline components/about-govtech/about-govtech.tsx:11]: There no reason for max-w-96 here?

File: components/about-govtech/about-govtech.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import { Button } from "../ui/button";
+import Text from "../ui/text";
+
+export default function AboutGovtech() {
+  return (
+    <div className=" mx-auto mt-5 p-5 md:flex">
+      <div className="max-w-md">
+        <Text variant="subheading" className="text-[#0F45A8]">
```
Comment (avevotsira) [inline components/about-govtech/about-govtech.tsx:8]: you don't color text like that , use one of the fore color

File: components/about-govtech/about-govtech.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+          PageMaker including versions of Lorem Ipsum.
+        </Text>
+        <Text variant="body" className="mt-12">
+          March 14, 2024
+        </Text>
+        <Text variant="body" className="mt-5 ">
+          It is a long established fact that a reader will be distracted by the readable content
+          of a page when looking at its layout
+        </Text>
+        <Button className="mt-8 ml-28 bg-primary text-white">View Agenda</Button>
```
Comment (avevotsira) [inline components/about-govtech/about-govtech.tsx:23]: why are we adjsuting this using margin , use a flex box

File: components/about-govtech/about-govtech.tsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+      </div>
+      <div className="mt-8 md:mt-0">
+        <img className="h-96 h-auto w-full max-w-3xl" src="HRTechx 1.png" alt="" />
```
Comment (avevotsira) [inline components/about-govtech/about-govtech.tsx:26]: use next image. what's the purpose of w-full max-w-3xl?

File: components/about-govtech/about-govtech.tsx [hunk 7]
```diff
@@ -0,0 +1,30 @@
+import { Button } from "../ui/button";
+import Text from "../ui/text";
+
+export default function AboutGovtech() {
+  return (
+    <div className=" mx-auto mt-5 p-5 md:flex">
```
Comment (avevotsira) [inline components/about-govtech/about-govtech.tsx:6]: why is this a md:flex?

File: components/about-govtech/about-govtech.tsx [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+      <div className="max-w-md">
```
Comment (avevotsira) [inline components/about-govtech/about-govtech.tsx:7]: what is this for?

**Review by Visethboti — APPROVED** [review]

**Pull request merged**